tests/helpers/fs.bash

```diff
@@ -85,3 +85,11 @@ function selinux_on() {
     return 1
   fi
 }
+
+# Returns linux distro running on the host (technically, in the test-priv container).
+function get_distro() {
```

Comment:
nit: get_distro -> get_host_distro()

Reply:
Sure. Technically, it's not the host-distro, that's why i didn't do it, but it's true that it can confuse reader thinking it refers to the sys-container distro.